Partition_2/doc/Partition_2/CGAL/partition_is_valid_2.h

```diff
@@ -134,9 +134,9 @@ A valid partition is one in
 which the polygons are nonoverlapping and the union of the polygons is the 
 same as the original polygon and each polygon is \f$ y\f$-monotone 
 
-\pre P
+\pre
```

Comment:
shall `\pre` be replaced by `\return`?

Reply:
We could, but then it should be a proper dixygenization as Partition_2 does not use `\tparam`, etc.

Reply:
I mean `\pre` is a precondition so simply removing should be enough.

Reply:
The text of the precondition follows in the next line.  The `P` was a leftover from a copy paste of another precondition.